etc/telegraf.conf

```diff
@@ -860,6 +860,18 @@
 #   ## Master node.
 #   cluster_stats = false
 #
+#   ## Set indices_stats to true when you want to obtain indices stats from the Master node.
+#   indices_stats = false
+#
+#   ## Set shards_stats to true when you want to obtain shards stats from the Master node.
+#   ## If set, then indices_stats is considered true as they are also provided with shard stats.
+#   shards_stats = false
+
+#   ## Multiplier of the elasticsearch interval to be used to reduce the frequency of
+#   ## indices_stats and shards_stats reports. The default interval is 10 seconds, and this
+#   ## multiplier is defaulted to 6 to cause these metrics to be taken once per minute by default.
+#   indices_interval_multiplier = 6
```

Comment:
why have you done this? is there a reason that these metrics need to be rate-limited?

If they do need to be rate-limited, then indicate why in the doc and the README. Also, I think it should be just a configurable interval (ie, "60s") rather than a multiplier.